server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/repository/Repository.java

```diff
@@ -0,0 +1,60 @@
+package de.tum.in.www1.hephaestus.codereview.repository;
+
+import java.time.Instant;
+import java.util.List;
+
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "repositories")
+@Getter
+@Setter
+@NoArgsConstructor
+public class Repository {
+
+    /**
+     * Unique identifier for a Repository
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "github_id")
+    private Long githubId;
+
+    @Column
```

Comment:
I think you can omit `@Column`, or you just always have it and fix the column name to some value.